examples/vgrade.py

```diff
@@ -95,6 +95,8 @@ def main():
         elif event.type == pg.USEREVENT:
             DisplayGradient(screen)
 
+    pg.quit()
```

Comment:
Should most of these examples just be try-finally constructions around the call to `main()`?

Reply:
Not sure what you mean...

```
try:
    main()
finally:
    pg.quit()
```
So that way, if it crashes halfway through it's guaranteed to call pg.quit()?

Reply:
Is consistency here good in general? Is this a pattern we want to promote? As far as I can tell, the PR adds pg.quit() in the correct place, which is sometimes after the call to main, sometimes at the end of main, sometimes at the end of a script with no main, and sometimes right before raising sysexit. It works, but it isn't pretty.